plugins/matplotlib/src/MatPlotLibPanel.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/no-unused-state */
+import React from "react";
+import "./MatPlotLibPanel.scss";
+
+export type MatPlotLibWidget = {
+  type: string;
+  getDataAsBase64: () => string;
+};
+
+export type MatPlotLibPanelProps = {
+  fetch: () => Promise<MatPlotLibWidget>;
+};
+
+export type MatPlotLibPanelState = {
+  imageData?: string;
+};
+
+/**
+ * Wraps and IrisGridPanel to add a refresh button for Pandas.
+ */
+export class MatPlotLibPanel extends React.Component<
```

Comment:
Any reason it's a class component instead of functional w/ hooks?

Reply:
I'll make a functional component, just copied from somewhere else.